src/visidentifier/vismessage.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <Poco/JSON/Parser.h>
+#include <Poco/JSON/Stringifier.h>
+#include <aos/common/tools/enum.hpp>
+
+#include "utils/exception.hpp"
+#include "utils/json.hpp"
+#include "vismessage.hpp"
+
+/***********************************************************************************************************************
+ * Public
+ **********************************************************************************************************************/
+
+VISMessage::VISMessage(const VISAction action)
+    : mAction(action)
+    , mJsonObject(new Poco::JSON::Object)
```

Comment:
should we deallocate memory? or maybe better to use json object type instead of PTR ?

Reply:
Applied your suggestion. Refactored to store object, not ptr